DISCOVER/13_taxonomy.md

```diff
@@ -1,17 +1,13 @@
----
-layout: page
-title: Taxonomy
----
-
+(topics)=
 # Taxonomy
```

Comment:
```suggestion
# Glossary
```

Reply:
I like Glossary too. There is a built in feature for Glossary in JBook / Sphinx. I think we can repurpose this page. Also the way the current taxonomy page is used is to collect the tags and redirect them. With #60 this automatically gets generated with Melissa's sphinx-tags tool, and we can name it anything we want, but by default it's named "Tags Overview"